src/styles/main.scss

```diff
@@ -0,0 +1,52 @@
+.watch {
+  position: relative;
+  margin: 200px auto;
+  height: 400px;
+  width: 400px;
+  border: 1px solid deepskyblue;
+  border-radius: 100%;
+
+  &__arrow {
+    position: absolute;
+    bottom: 50%;
+    left: 49%;
```

Comment:
Why 49% ?
What if other proportions will change?

Reply:
i search center in the circle